backend/sirius-components-collaborative-diagrams/src/main/resources/schema/diagram.graphqls

```diff
@@ -306,7 +306,7 @@ union InvokeNodeToolOnDiagramPayload = ErrorPayload | InvokeNodeToolOnDiagramSuc
 
 type InvokeNodeToolOnDiagramSuccessPayload {
   id: ID!
-  diagram: Diagram!
+  newSelection: WorkbenchSelection!
```

Comment:
The GraphQL schema of the `InvokeEdgeToolOnDiagramSuccessPayload` should be updated too (I'll do it)

Reply:
Done